coral/src/app/pages/configuration/environments/index.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { screen } from "@testing-library/react/pure";
+import EnvironmentsPage from "src/app/pages/configuration/environments";
+import { customRender } from "src/services/test-utils/render-with-wrappers";
+
+describe("Environments page", () => {
+  describe("renders Environments page with correct text", () => {
+    beforeAll(() => {
```

Comment:
cleanup is missing in afterAll. Not important now, just to avoid potential confusion in case another block gets added somewhat in the future.

Reply:
[9b7b13e](https://github.com/Aiven-Open/klaw/pull/1720/commits/9b7b13e2d0744c950bf8b8e163188957ddf6f3b1)